.github/workflows/main.yml

```diff
@@ -108,11 +114,70 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: hadolint/hadolint-action@f988afea3da57ee48710a9795b6bb677cc901183  # v2.1.0
+        with:
+          dockerfile: docker/Dockerfile
+
+  docker-smoketest:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: jobserver
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run prod image and see if it can serve a basic test
+        env:
+          DOCKER_BUILDKIT: 1
+          COMPOSE_DOCKER_CLI_BUILD: 1
+          COMPOSE_PROJECT_NAME: job-server
+        run: |
+          cat >> .env <<EOL
+          DATABASE_URL=postgres://user:password@172.17.0.1/jobserver
+          DJANGO_VITE_DEV_MODE=False
+          GITHUB_TOKEN=
+          SECRET_KEY=12345
+          SOCIAL_AUTH_GITHUB_KEY=test
+          SOCIAL_AUTH_GITHUB_SECRET=test
+          EOL
+          docker-compose -f docker/docker-compose.yaml up --detach prod
```

Comment:
This would work when running locally, but it would breaking in production

The issue is that gunicorn is misconfigured - it's listening on the default of 127.0.0.1, rather than 0.0.0.0

This is likely a consequence of moving away from dokku's magic Procfile config, which whilst looked similar, and used our gunicorn config, it somehow still worked.

Reply:
The fix is to add `bind = "0.0.0.0"` to gunicorn.conf.py

Reply:
Then our prod image works the same whether run locally or in dokku